runtime/doc/lua.txt

```diff
@@ -47,20 +47,20 @@ the existing `package.cpath` are used.  Example:
 
 1. Given that
    - 'runtimepath' contains `/foo/bar,/xxx;yyy/baz,/abc`;
-   - initial (defined at compile-time or derived from 
```

Comment:
Can you remove the whitespace changes from the doc files?

Reply:
I've done that in a separate commit. Is that ok or does it need its own PR? Or is that whitespace intentionally there?

Reply:
Yeah, avoid unrelated formatting changes -- makes it harder to review (and may be clobbered by autogenerated docs anyway).

Reply:
Got it.